index.js

```diff
@@ -1,8 +1,14 @@
 const util = require('util')
-const chalk = require('chalk')
+
+const chalk = {
+  red: (str) => `\u001b[31m${str}\u001b[0m`,
+  gray: (str) => `\u001b[90m${str}\u001b[0m`,
+  white: (str) => `\u001b[37m${str}\u001b[0m`,
+  bold: (str) => `\u001b[1m${str}\u001b[22m`
```

Comment:
tests in `chalk` repo which confirms current implementation is correct https://github.com/chalk/chalk/blob/f7b29ae8ef4fd2048e08aa361778d290ed10ce7a/test/chalk.js#L23C7-L23C7

Reply:
Thanks, after looking at it, I've changed the closing sequence to match what was used by Chalk. (from 0m to 39m)